exports.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
 
     'configServiceEnabled'          : require(__dirname + '/plugins/configservice/configServiceEnabled.js'),
 
+    'asgMultiAz'                    : require(__dirname + '/plugins/ec2/asgMultiAz.js'),
```

Comment:
Please create a new folder for `autoscaling` in `plugins`: 
`'asgMultiAz'                    : require(__dirname + '/plugins/autoscaling/asgMultiAz.js'),`